modules/olm/crds.tf

```diff
@@ -0,0 +1,17 @@
+module "yaml" {
+  source = "../../../terraform-null-helpers/modules/yaml"
+  # source =
+
+  files = [
+    "https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/v0.25.0/deploy/upstream/quickstart/crds.yaml",
+    "https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/v0.25.0/deploy/upstream/quickstart/olm.yaml"
+  ]
+}
+
+resource "kubernetes_manifest" "olm" {
+  for_each = toset(module.yaml.yamls)
+
+  # provider = kubernetes-alpha
```

Comment:
please remove commented unused codes

Reply:
This is still experimental, please do not pay attention, those changes are more related to node-problem-detector than OLM.